### DISCUSSION TITLE
Reference paper for each of the environments

### DISCUSSION THREAD

**Discussion by ReHoss** [dynamicslab/hydrogym#72] [Q&A]
Hi everyone,

Could you provide references of papers which give details on the problems/environments configurations ?

Like for the cavity flow, @jcallaham gave the reference (Sipp et al.) in some Github issue.

Could you provide one reference for each of the flow environment please, later to be incorporated in the documentation (I could do it if you want). Such that anyone can rely on those refs to better understand the features of the flows.

Thanks a lot for your efforts:)

**Comment by jcallaham** [comment]
Good idea!  How does this look to you?

https://github.com/dynamicslab/hydrogym/blob/main/docs/source/quickstart.rst

**Reply by ReHoss** [reply]
> Good idea! How does this look to you?
> 
> https://github.com/dynamicslab/hydrogym/blob/main/docs/source/quickstart.rst

the link is dead:/
